src/main/webapp/resources/js/pages/projects/samples/components/CreateNewSample.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { AutoComplete, Form, Input, Modal } from "antd";
+import {
+  createNewSample,
+  validateSampleName,
+} from "../../../../apis/projects/samples";
+import searchOntology from "../../../../apis/ontology/taxonomy/query";
+
+/**
+ * React component to create a new sample within a project.
+ * @returns {JSX.Element}
```

Comment:
Add params/functions in the jsdoc

Reply:
Don't think that is really necessary as it is the default form API from Ant Design as I have already stated in the JSDoc.  Any need references should be looked at there.